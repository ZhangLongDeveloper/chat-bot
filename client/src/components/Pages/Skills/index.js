import React from 'react';

import EmptyView from '../../Common/EmptyView';

const Skills = () => (
  <div>
    <EmptyView section="skills" />
  </div>
);

export default Skills;
