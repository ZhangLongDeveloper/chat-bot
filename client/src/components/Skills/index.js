import React from 'react';

const Skills = () => (
  <div>
    This is SKILLS
  </div>
);

export default Skills;
