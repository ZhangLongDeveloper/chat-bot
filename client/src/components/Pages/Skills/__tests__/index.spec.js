import React from 'react';
import { create } from 'react-test-renderer';

import Skills from '../../Skills';

jest.mock('../../../Common/EmptyView', () => 'EmptyView');

const mockComponent = props => (
  <Skills {...props} />
);

describe('components/Pages/<Skills />', () => {
  it('should render component', () => {
    const tree = create(mockComponent()).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
