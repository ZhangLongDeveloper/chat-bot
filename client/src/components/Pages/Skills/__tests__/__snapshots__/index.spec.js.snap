// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Pages/<Skills /> should render component 1`] = `
<div>
  <EmptyView
    section="skills"
  />
</div>
`;
